### ISSUE TITLE
SSL Issues

### ISSUE THREAD

**Issue by basex** [laurilehmijoki/s3_website#198]
On every deploy I get for every change:

```shell
[wait] Failed to upload contact (Failure initializing default SSL context). Trying again in 8 seconds.
[fail] Failed to upload contact (Failure initializing default SSL context)
[fail] Failed to push the website to http://...
```

Any idea what might be?

OS: OSX 10.11.1
Ruby: 2.2.3
Java: 1.8.0_25

**Comment by basex** [comment]
Solved problem by upgrading to Java 1.8.0_66 and using jenv.

**Comment by laurilehmijoki** [comment]
Thanks for sharing your solution.
